packages/@react-aria/utils/src/useViewportSize.ts

```diff
@@ -21,7 +22,8 @@ interface ViewportSize {
 let visualViewport = typeof document !== 'undefined' && window.visualViewport;
 
 export function useViewportSize(): ViewportSize {
-  let [size, setSize] = useState(() => getViewportSize());
+  let isSSR = useIsSSR();
+  let [size, setSize] = useState(() => isSSR ? {width: 0, height: 0} : getViewportSize());
```

Comment:
I think to add `isSSR` check inside `getViewportSize`, but this function also used in `useEffect` hook (I would have to add extra deps to hook)

Reply:
This seems fine, the `useEffect` won't run on the server anyways.